pkg/sdk/testint/grants_integration_test.go

```diff
@@ -254,7 +258,31 @@ func TestInt_GrantAndRevokePrivilegesToAccountRole(t *testing.T) {
 			},
 		}
 		err := client.Grants.GrantPrivilegesToAccountRole(ctx, privileges, on, roleTest.ID(), nil)
-		require.ErrorContains(t, err, "unexpected 'SEARCH'")
+		require.NoError(t, err)
+
+		grants, err := client.Grants.Show(ctx, &sdk.ShowGrantOptions{
+			To: &sdk.ShowGrantsTo{
+				Role: roleTest.ID(),
+			},
+		})
+		require.NoError(t, err)
+		selectPrivilege, err := collections.FindOne[sdk.Grant](grants, func(g sdk.Grant) bool {
```

Comment:
nit; variable suggests select privilege but usage is provided

Reply:
ugh, copy-pasted, will change in the next PR, oky?

Reply:
or not, one test more failed, so fixing now